src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/ResourceController.cs

```diff
@@ -149,5 +149,90 @@ public ActionResult LayoutSettings(string org, string app)
             string settings = _appResourceService.GetLayoutSettings();
             return Ok(settings);
         }
+
+        /// <summary>
+        /// Get the layout settings.
+        /// </summary>
+        /// <param name="org">The application owner short name</param>
+        /// <param name="app">The application name</param>
+        /// <returns>The settings in the form of a string.</returns>
+        [HttpGet]
+        [Route("{org}/{app}/api/layoutsets")]
+        public ActionResult LayoutSets(string org, string app)
```

Comment:
I think we should name methods normally. In this case: `GetLayoutSets`. There are no technical reasons to give the method a name that match a path part from the route.

Reply:
Fixed